app/views/instructeur_mailer/send_notifications.html.haml

```diff
@@ -4,7 +4,7 @@
   Bonjour,
 
 %p
-  Vous avez du nouveau sur #{APPLICATION_NAME}.
+  Vous avez du nouveau sur #{APPLICATION_NAME} depuis hier.
```

Comment:
elle est envoyé le we ?

Reply:
@LeSim non... Je fais un p'tit commit pour traiter ce cas particulier

Reply:
Est-ce qu'avec ce commit, tu préfères @LeSim ?
https://github.com/betagouv/demarches-simplifiees.fr/pull/5770/commits/5ffe0f618fbf6d287519dbd08df9e3612f7416ed